<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingDialog</class>
 <widget class="QDialog" name="SettingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>356</width>
    <height>88</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>20</y>
     <width>81</width>
     <height>241</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QRadioButton" name="normal_radio">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Normal</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="largescale_radio">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>81</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>LargeScale</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>45</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Player1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>50</y>
     <width>45</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Player2</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_Player0">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>20</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Human</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Computer</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_Player1">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>50</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Human</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Computer</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
